stacks-signer/src/client/stacks_client.rs

```diff
@@ -78,6 +78,7 @@ pub struct StacksClient {
 
 #[derive(Deserialize)]
 struct GetStackersErrorResp {
+    #[allow(dead_code)]
     err_type: String,
```

Comment:
Seems like no one uses this now. Let's just delete it?

Reply:
It will not deserialize correctly if i do that. Though I could deserialize the single field altrnatively.